File: TransCoder_tokenized_test_set_functions/python/CHECK_HALF_STRING_CHARACTER_FREQUENCY_CHARACTER.py
def checkCorrectOrNot ( s ) :
    global MAX_CHAR
    count1 = [ 0 ] * MAX_CHAR
    count2 = [ 0 ] * MAX_CHAR
    n = len ( s )
    if n == 1 :
        return true
    i = 0;
    j = n - 1
    while ( i < j ) :
        count1 [ ord ( s [ i ] ) - ord ( 'a' ) ] += 1
        count2 [ ord ( s [ j ] ) - ord ( 'a' ) ] += 1
        i += 1;
        j -= 1
    for i in range ( MAX_CHAR ) :
        if count1 [ i ] != count2 [ i ] :
            return False
    return True

File: TransCoder_tokenized_test_set_functions/cpp/CHECK_HALF_STRING_CHARACTER_FREQUENCY_CHARACTER.cpp
bool checkCorrectOrNot ( string s ) {
  int count1 [ MAX_CHAR ] = {
    0 };
    int count2 [ MAX_CHAR ] = {
      0 };
      int n = s . length ( );
      if ( n == 1 ) return true;
      for ( int i = 0, j = n - 1;
      i < j;
      i ++, j -- ) {
        count1 [ s [ i ] - 'a' ] ++;
        count2 [ s [ j ] - 'a' ] ++;
      }
      for ( int i = 0;
      i < MAX_CHAR;
      i ++ ) if ( count1 [ i ] != count2 [ i ] ) return false;
      return true;
    }